frigate/embeddings/onnx/lpr_embedding.py

```diff
@@ -261,8 +261,10 @@ def _load_model_and_utils(self):
     def _preprocess_inputs(self, raw_inputs):
         if isinstance(raw_inputs, list):
             raise ValueError("License plate embedding does not support batch inputs.")
-        # Get image as numpy array
-        img = self._process_image(raw_inputs)
+
+        pil = self._process_image(raw_inputs)
```

Comment:
why not just never call _process_image in the first place? if you want an np array better to not double convert sicne that is all process image does

Reply:
That's fair, and definitely a better approach